src/App.tsx

```diff
@@ -1,16 +1,23 @@
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+export const App:React.FC = () => {
+  const [movies, setMovies] = useState([...moviesFromServer]);
+  const onAdd = (movie: Movie) => {
+    setMovies(prevMovies => [...prevMovies, movie]);
+  };
```

Comment:
Better add 1 empty line between logic blocks of code 
```suggestion
  const [movies, setMovies] = useState([...moviesFromServer]);
  
  const onAdd = (movie: Movie) => {
    setMovies(prevMovies => [...prevMovies, movie]);
  };
```